### DISCUSSION TITLE
Feature Request: Default language for script tag

### DISCUSSION THREAD

**Discussion by luc122c** [nuxtrdev/nuxtr-vscode#23] [Ideas]
Hi There! 

The 'Default Language' setting for style tags is great. It would be helpful to have the same for script tags. 

For example, 
`<script setup lang="ts"> </script>`

**Comment by adhamfarrag** [comment]
Hey @luc122c!

We will add it to the next release. :)

**Reply by luc122c** [reply]
That's amazing. Thanks so much :)
